ApexDocContent/API.htm

```diff
@@ -0,0 +1,21 @@
+<html>
+	<body>
+		<h2>Nonprofit Starter Pack API (Application Public Interface)</h2>
```

Comment:
Regarding the question about build status, we should wait for the next
build to complete before merging a branch unless there is a reason we need
to rush the process for a hot fix or something.

The reason is that the build that's pending is to test if the feature
branch you want to merge works with the code that was just merged into
dev.  It's rare to see failures there but it's better to wait to make sure.

On Fri, Jan 23, 2015 at 12:01 PM, Nicolas Campbell <notifications@github.com
> wrote:

> In ApexDocContent/API.htm
> <https://github.com/SalesforceFoundation/Cumulus/pull/1242#discussion_r23463507>
> :
>
> > @@ -0,0 +1,21 @@
> > +<html>
> > +	<body>
> > +		<h2>Nonprofit Starter Pack API (Application Public Interface)</h2>
>
> Isn't it Application Programming Interface?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SalesforceFoundation/Cumulus/pull/1242/files#r23463507>
> .
>